doc/userguide/configuration/multi-tenant.rst

```diff
@@ -55,6 +73,33 @@ Settings:
     - vlan-id: 1112
       tenant-id: 3
 
+
+This example uses the VLAN tuple selector for packets encapsulated with two VLAN ids:
```

Comment:
I'm still a bit confused with this part, it mentions two VLAN ids, but the vlan-tuple examples have three ids, maybe this is just because I don't understand how vland ids work, or do you think the part can be reworded?